hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/endpoint/BrokerOperations.java

```diff
@@ -39,8 +42,11 @@ public BrokerOperations(Map<String, String> kafkaZkConnection, ConfigFactory con
     public BrokerOperations(Map<String, String> kafkaZkConnection, int sessionTimeout, int connectionTimeout, String namespace) {
         zkClients = kafkaZkConnection.entrySet().stream()
                 .collect(toMap(Map.Entry::getKey,
-                               e -> new ZkClient(e.getValue(), sessionTimeout, connectionTimeout, ZKStringSerializer$.MODULE$)));
-
+                               e -> {
+                                   ZkConnection connection = new ZkConnection(e.getValue(), sessionTimeout);
+                                   ZkClient zkClient = new ZkClient(connection, connectionTimeout, ZKStringSerializer$.MODULE$);
+                                   return new ZkUtils(zkClient, connection, false);
```

Comment:
Sorry, wasn't clear - I meant everything but security, is it considered to be functionally correct in PR as it is now? If so, we'd be happy to test security and performance as well.